app/models/zone.rb

```diff
@@ -4,10 +4,12 @@ class Zone < ApplicationRecord
 
   serialize :settings, Hash
 
-  belongs_to      :log_file_depot, :class_name => "FileDepot"
+  belongs_to :log_file_depot, :class_name => "FileDepot"
 
   has_many :miq_servers
   has_many :ext_management_systems
+  has_many :paused_ext_management_systems, :class_name => 'ExtManagementSystem', :inverse_of => :zone_before_pause
```

Comment:
Does this work?  Do you have to tell it which column to use for this relation?

Reply:
If the assoc name doesn't match the foreign key you need to specify it:

```
>> ExtManagementSystem.first.pause!
=> true
>> Zone.first.paused_ext_management_systems
=> #<ActiveRecord::Associations::CollectionProxy []>
```

But when I add `:foreign_key => :zone_before_pause_id` then it works:
```
>> Zone.first.paused_ext_management_systems
=> #<ActiveRecord::Associations::CollectionProxy [#<ManageIQ::Providers::Openshift::ContainerManager id: 1, ....
```

Reply:
@agrare 
ahh, you're right, but what's purpose of `:inverse_of` then? 
It should search for foreign key in bi-directionals associations and belongs_to's foreign_key corresponds to belongs_to name. (zone_before_pause => zone_before_pause_id)
https://guides.rubyonrails.org/association_basics.html#bi-directional-associations

Reply:
it seems like a bug in inverse_of for me, or not?